ArmVirtPkg/ArmVirt.dsc.inc

```diff
@@ -269,6 +264,18 @@
 [LibraryClasses.ARM]
   ArmSoftFloatLib|ArmPkg/Library/ArmSoftFloatLib/ArmSoftFloatLib.inf
   RngLib|MdePkg/Library/BaseRngLibTimerLib/BaseRngLibTimerLib.inf
+!if $(NETWORK_TLS_ENABLE) == TRUE
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLib.inf
+!else
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
+!endif
+
+[LibraryClasses.AARCH64]
+!if $(NETWORK_TLS_ENABLE) == TRUE
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLibFullAccel.inf

```

Comment:
For the record, `TlsLib.inf` effectively uses the Ecc function `EC_KEY_new_by_curve_name()`,
so it makes sense to use the Ful version of the OpensslLib instead of the previous `OpensslLib.inf` when `NETWORK_TLS_ENABLE=TRUE`.

However it seems to make some build fail due to the size increase